src/main/java/edu/si/fcrepo/ObjectProcessor.java

```diff
@@ -156,8 +167,9 @@ public void accept(final URI objectId) {
                     final SAXParser parser = saxFactory.newSAXParser();
                     parser.parse(dcXML, new DublinCoreContentHandler(tripleSink, createURI(objectId.toString())));
                 });
-                if (relsIntDatastream != null)
+                if (relsIntDatastream != null) {
```

Comment:
No more braces!

Reply:
How do I turn this off in Eclipse. I set up new formatting rules and everything but it keeps adding them. 😠

Reply:
Are you using the `Format` command? That's what will trigger the rules. But don't worry about it a lot, if it's not obvious we'll merge and I'll fiddle with it later.